_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -0,0 +1,230 @@
+
+- You must install and configure [gcloud command line tool](https://cloud.google.com/sdk/docs/){:target="_blank"} and include the ```kubectl``` component (```gcloud components install kubectl```).  If you do not wish to install the gcloud client, you can use [Google Cloud Shell](https://cloud.google.com/shell/docs/){:target="_blank"} to perform these tasks.
+
+- <img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning" style="width: 32px; display:inline" /> You must set your default compute service account to include:
+> - ```roles/container.admin```  (Kubernetes Engine Admin)
+> - ```Editor```  (on by default)
+
+To set this, navigate to the IAM section of the [Cloud Console](https://console.cloud.google.com/iam-admin/iam/project){:target="_blank"} and find your default GCE/GKE service account in the following form: `projectNumber-compute@developer.gserviceaccount.com`  then under **Roles**, find the ```Kubernetes Engine``` group and set the role ```Kubernetes Engine Admin```.
+
+
+## Setup
+
+### Launch Deployment Manager
+
+Once you have an account and project enabled, click the following link to open the Deployment Manager.
+
+- [Istio GKE Deployment
+  Manager](https://accounts.google.com/signin/v2/identifier?service=cloudconsole&continue=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&followup=https://console.cloud.google.com/launcher/config?templateurl=https://raw.githubusercontent.com/istio/istio/master/install/gcp/deployment_manager/istio-cluster.jinja&flowName=GlifWebSignIn&flowEntry=ServiceLogin){:target="_blank"}
+
+We recommend that you leave the default settings as the rest of this tutorial shows how to access the installed features. By default the tool creates a GKE alpha cluster with the specified settings, then installs the Istio [control plane](https://istio.io/docs/concepts/what-is-istio/overview.html#architecture){:target="_blank"}, the [BookInfo](https://istio.io/docs/guides/bookinfo.html){:target="_blank"} sample app, [Grafana](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html){:target="_blank"} with [Prometheus](https://istio.io/docs/tasks/telemetry/querying-metrics.html){:target="_blank"}, [ServiceGraph](https://istio.io/docs/tasks/telemetry/servicegraph.html){:target="_blank"}, and [Zipkin](https://istio.io/docs/tasks/telemetry/distributed-tracing.html#zipkin){:target="_blank"}. You'll find out more about how to access all of these below.
+
+Click ```Deploy```
+
+![GKE-Istio Launcher](img/dm_launcher.png)
+
+Wait until Istio is fully deployed. Note that this can take up to five minutes.
+
+### Bootstrap gcloud
+
+Once deployment is complete, do the following on the workstation where you've installed `gcloud`:
+
+* Bootstrap kubectl for the cluster you just created and confirm the cluster is
```

Comment:
Should be a numbered list (1.) rather than an unordered list (*)

Reply:
done; 
( finally realized you can indent code blocks into the numbered list!)